config/ci/manager/manager_pull_policy.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     spec:
       containers:
       - name: manager
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
this change seems unrelated

Reply:
Good point regarding documentation for providers.

At the very least, I think we should test this with #3985

Reply:
This is changed during the build (Makefile)